tests/View/ViewBladeCompilerTest.php

```diff
@@ -207,6 +207,30 @@ public function testCommentsAreCompiled()
 this is a comment
 */ ?>';
         $this->assertEquals($expected, $compiler->compileString($string));
+
+        $string = '{{--
+this is an extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely long comment
+--}}';
+        $expected = '<?php /*
+this is an extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely
+extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely extremely long comment
+*/ ?>';
+        $this->assertEquals($expected, $compiler->compileString($string));
```

Comment:
As far as I know the line numbers will already be different slightly due to the `extends` syntax, sections, etc... I personally think comments should be entirely removed.